Hypercube/src/Controller/HypercubeController.php

```diff
@@ -78,7 +78,7 @@ public function get(Request $request)
         $this->log->debug("Got Content-Type:", ['type' => $content_type]);
 
         if ($content_type == 'application/pdf') {
-            $cmd_string = $this->pdftotext_executable . " $args - -";
+            $cmd_string = $this->pdftotext_executable . " $args -enc UTF-8 - -";
```

Comment:
Seems to make sense. Reading through `pdftotext`'s docs any idea why they defaulted to `Latin1`.

Reply:
I went back to look and this appears to be a version issue. Older versions default to Latin1, probably due to not paying attention to Unicode, while the newer versions default to UTF-8.

Oddly, the man pages for a default site say UTF-8 is the default, but running the version flag on it shows an older version that requires the flag.

Reply:
Had a peak through about available encodings and what may or may not be available on the OS. Know we've ran into things in PHP where things like LC_ALL and UTF-8 aren't available but I can't seem to find reference to that in the `pdftotext` context so this gets a 👍  from me.